_data/main.yaml

```diff
@@ -125,3 +125,19 @@ events:
     begin: 2023-09-05 09:00:00
     duration: { days: 2 }
     event_url: https://rsecon23.society-rse.org/
+
+  - summary: 18th HiRSE Seminar - Neil Chue Hong: Can Software Metrics Improve Software Quality?
```

Comment:
```suggestion
  - summary: "18th HiRSE Seminar - Neil Chue Hong: Can Software Metrics Improve Software Quality?"
```

Have to do this to get yaml parsing to behave

Reply:
Sorry, I know these things usually. Was just copy-pasting something together. Although some general advice is probably a good idea.